src/asset/asset-localized.js

```diff
@@ -0,0 +1,202 @@
+Object.assign(pc, function (){
+    var LocalizedAsset = function (app) {
+        this._app = app;
+        app.i18n.on('set:locale', this._onSetLocale, this);
+
+        this._autoLoad = false;
+        this._disableLocalization = false;
+
+        this._defaultAsset = null;
+        this._localizedAsset = null;
+
+        pc.events.attach(this);
+    };
+
+    LocalizedAsset.prototype._bindDefaultAsset = function () {
+        var asset = this._app.assets.get(this._defaultAsset);
+        if (!asset) return;
```

Comment:
Looks like we no longer handle the case where the asset is added to the registry after the element is created?

Reply:
We handle it in onSetLocale but we do not handle adding the defaultAsset later when it comes to the set:localized and remove:localized events which I'll fix. Basically whatever ends up in localizedAsset is the asset where we listen for load events etc.